pkgs/os-specific/linux/input-utils/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "input-utils-${version}";
+  version = "1.3";
+  
+  src = fetchurl {
+    url = "https://www.kraxel.org/releases/input/input-${version}.tar.gz";
+    sha256 = "11w0pp20knx6qpgzmawdbk1nj2z3fzp8yd6nag6s8bcga16w6hli";
+  };
+
+  prePatch = ''
+    # Adds full path to "linux/input.h"
+    substituteInPlace ./emulate-key.c --replace "linux/input.h" "${stdenv.glibc.dev}/include/linux/input.h"
+    substituteInPlace ./input.h       --replace "linux/input.h" "${stdenv.glibc.dev}/include/linux/input.h"
+    # Use proper include path for kernel include files.
+    substituteInPlace ./name.sh       --replace "/usr/include/linux/" "${stdenv.glibc.dev}/include/linux/"
```

Comment:
Make sure you have an assertion for stdenv without glibc. Just do something like:

assert stdenv ? glibc;